tests/FormTest/packages.config

```diff
@@ -1,10 +1,16 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Autofac" version="4.6.2" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.4" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.4" targetFramework="net461" />
-  <package id="Microsoft.Net.Compilers" version="1.2.1" targetFramework="net46" developmentDependency="true" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.2" targetFramework="net46" />
-  <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.2.1" targetFramework="net46" />
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Autofac" version="4.2.0" targetFramework="net461" />

```

Comment:
Why is this using AutoFac 4.2.0, but the previous version was using 4.6.2? Is downgrading intentional?